src/formatDate.js

```diff
@@ -8,7 +8,35 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat.pop();
```

Comment:
Consider using the spread operator to clone the `fromFormat` and `toFormat` arrays instead of `pop()`, which mutates the original arrays. This can lead to unexpected behavior if the `formatDate` function is called multiple times with the same format arrays.